### PR TITLE
66 style Home, Register and Login pages 

### PR DESCRIPTION
**Pull request by AL-ASHMALI**
## Naming Rules
style fix - Rabie Al-Ashmali

## Description

There were some issues with the style of the home, register and login page. 

## Related to

#66 

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/Login.js [hunk 1]
```diff
@@ -69,22 +70,15 @@ const Login = () => {
...
-      <Form
-        noValidate
-        validated={validated}
-        onSubmit={handleChange}
-        style={{ width: "50%", margin: "auto" }}
-      >
-        <Form.Group as={Col} className="mb-4" controlId="formBasicEmail">
-          <Form.Label className="text-light">Email</Form.Label>
+    <div className="container">
+      <h1>LOGIN</h1>
```
Comment (RainbowGiantSquid) [inline client/src/components/Login.js]: could this be using the heading component? Same as the register page please.

File: client/src/components/Register.js [hunk 2]
```diff
@@ -85,20 +85,19 @@ const Register = () => {
       <div className="container">
         <h1>Register</h1>
```
Comment (RainbowGiantSquid) [inline client/src/components/Register.js]: could we please use a heading component here as well?

File: client/src/components/Register.js [hunk 3]
```diff
@@ -131,7 +130,7 @@ const Register = () => {
                 <Form.Control
                   className="input"
                   type={passwordShown ? "text" : "password"}
-                  placeholder="password"
+                  placeholder="Password..."
```
Comment (RainbowGiantSquid) [inline client/src/components/Register.js:134]: nice, thanks for updating placeholders

**Review by RainbowGiantSquid — CHANGES_REQUESTED** [review]

File: client/src/components/Login.js (hunk 1 above)
Comment (AL-ASHMALI) [reply client/src/components/Login.js]: Done

File: client/src/components/Register.js (hunk 3 above)
Comment (AL-ASHMALI) [reply client/src/components/Register.js:134]: 👍

**Review by RainbowGiantSquid — APPROVED** [review]

**Pull request merged**